.isort.cfg

```
@@ -6,7 +6,7 @@ import_heading_stdlib=Standard Python Libraries
 import_heading_thirdparty=Third-Party Libraries
 import_heading_firstparty=cisagov Libraries
 
-# Should be auto-populated by seed-isort-config hook
-known_third_party=boto3,botocore,bson,dateutil,docopt,gnupg,netaddr,pytest,pytz,requests,requests_aws4auth,setuptools
-# These must be manually set to correctly separate them from third party libraries
-known_first_party=dmarc,mongo_db_from_config
+known_first_party=dmarc, mongo_db_from_config
```

This is to (hopefuly) force the Python 2 version of this stdlib module to be treated as such (since `isort` is running under Python 3).

```suggestion
extra_standard_library=ConfigParser
known_first_party=dmarc, mongo_db_from_config
```

---

Good catch - taken care of in https://github.com/cisagov/cyhy-feeds/pull/42/commits/6e3754060c4fec84032a652c1c28caa1ceee4c7b.